iohk-monitoring/src/Cardano/BM/Backend/Switchboard.lhs

```diff
@@ -63,20 +63,31 @@ We are using an |MVar| because we spawn a set of backends that may try to send m
 the switchboard before it is completely setup.
 
 \begin{code}
+
 type SwitchboardMVar a = MVar (SwitchboardInternal a)
+
 newtype Switchboard a = Switchboard
-    { getSB :: SwitchboardMVar a }
+    { getSB :: SwitchboardMVar a 
+    }
 
 data SwitchboardInternal a = SwitchboardInternal
     { sbQueue     :: TBQ.TBQueue (LogObject a)
     , sbDispatch  :: Async.Async ()
     , sbLogBuffer :: !(Cardano.BM.Backend.LogBuffer.LogBuffer a)
     , sbLogBE     :: !(Cardano.BM.Backend.Log.Log a)
     , sbBackends  :: NamedBackends a
+    , sbRunning   :: !SwitchboardStatus
     }
 
 type NamedBackends a = [(BackendKind, Backend a)]
 
+-- | A bool isomorphic status for the switchboard
+-- which we can use.
```

Comment:
what does "which we can use" mean? the comment is not needed IMO.

Reply:
Meaning we can use it as a bool isomorphic status.

Reply:
This _can_ be removed if you think it's pointless.